biotools.py

```diff
@@ -0,0 +1,53 @@
+import sys
+import modules.dna_rna_tools as dr
+import modules.protein_tool as p
+import modules.fastqc_filter as f
+from typing import Dict, List, Union
```

Comment:
Все супер, только вот импорты из стандартной библиотеки всегда идут до импортов собственной библиотеки и разделяются пустыми строками

Плюс я была бы поосторожнее с собственными именами пакетов вроде dr, p, f - они хоть и могут быть короче, но лучше, чтобы оставались уникальными и информативными
```suggestion
import sys
from typing import Dict, List, Union


import modules.dna_rna_tools as dr
import modules.protein_tool as p
import modules.fastqc_filter as f

```